src/pages/roadmap.md

```diff
@@ -6,42 +6,43 @@ hide_table_of_contents: true
 
 Timelines are estimates and subject to change. For information about delivered features and fixes, see the [Changelog](/docs/changelog).
 
-## Release cycle
+## 2023 milestones
 
-We typically follow this release cycle:
+### Q4
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Allow/denylist user preferences available in all SDKs
 
-## Features roadmap
+## 2024 milestones
 
-### 2023 Q3
+### Q1
 
-- Onchain transaction reference content type
-- React SDK in Production
-- React Native SDK in Production (currently in Beta)
-- Android SDK in Production (currently in Beta)
-- XMTP v3 in Alpha
+- [Group chat via Messaging Layer Security (MLS)](https://github.com/xmtp/roadmap/milestone/1) in alpha
+- [Contact directory via did:ethr](https://github.com/xmtp/roadmap/milestone/5) in alpha
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/6) in alpha
 
-### 2023 Q4
+### Q2
 
-- XMTP v3 in Beta
+- [Group chat via MLS](https://github.com/xmtp/roadmap/milestone/2) in production
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/6) in production
+- [1:1 chats via MLS](https://github.com/xmtp/roadmap/milestone/3) in production
 
-### 2024 Q1
+### Q3
 
-- XMTP v3 in Production
-- Support for smart contract wallets
-- Portable preferences in Alpha - blocklists, allowlists, etc.
+- [XMTP fully using MLS](https://github.com/xmtp/roadmap/milestone/4) in production
+- [XMTP mainnet](https://github.com/xmtp/roadmap/milestone/8) in alpha
```

Comment:
Should we say a little bit about what mainnet is for? Is this a decentralized mainnet?